### ISSUE TITLE
Delete Old Files From Old Functionality

### ISSUE THREAD

**Issue by rachaelharris** [Allegheny-Ethical-CS/FakeNewsDetection#7]
Delete hard-coded csv files and other files from previous 2020 functionality that is no longer needed

**Comment by alimustafashah** [comment]
Can you assign this task to me?

**Comment by pmkuny** [comment]
Hi @rachaelharris  - I don't see a way to identify 2020 entries and/or files in the CSV files I found.  Could you point me in the right direction?

**Comment by rachaelharris** [comment]
@pmkuny yeah! I'm waiting on a push from another team member on this project. You are correct that sentiment.py is still using .csv files, but main.py will soon be updated to reflect new changes with api functionality. My team members will be taking care of this issue once we push our most recent code!

**Comment by pmkuny** [comment]
Sounds good, thanks for the update!

**Comment by rachaelharris** [comment]
Done!

**Comment by rachaelharris** [comment]
Done!
